src/Stratis.Bitcoin.Tests.Wallet.Common/WalletTestsHelpers.cs

```diff
@@ -113,6 +113,18 @@ public static (ChainedHeader ChainedHeader, Block Block) AppendBlock(Network net
             return (last, block);
         }
 
+        public static TransactionBuildOptions CreateOptions(WalletAccountReference accountReference, string password,
```

Comment:
I don't like the idea of dumping another method in this already massively bloated "Helper" class.